drivers/docker/driver.go

```diff
@@ -817,6 +817,13 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		Runtime: containerRuntime,
 	}
 
+	// This translates to docker create/run --cpuset-cpus option.
+	// --cpuset-cpus limit the specific CPUs or cores a container can use.
+	if driverConfig.CPUSetCPUs != "" {
+		logger.Debug(fmt.Sprintf("Setting CPUSetCPUs to %s", driverConfig.CPUSetCPUs))
```

Comment:
Let's remove this log line. It'll get pretty noisy and doesn't have a lot of added value once you're done developing the feature.

Reply:
Fixed.